### ISSUE TITLE
Duplication of find_ca function

### ISSUE THREAD

**Issue by j4real2208** [strimzi/strimzi-kafka-operator#6803]
At the current moment I believe the `find_ca` function is loaded from the `docker-images/kafka-based/kafka/scripts/tls_utils.sh` file. Therefore it seems like we to need to refactor the code in `kafka_tls_prepare_certificates.sh` to reflect the same as it is a duplicate effort to define it again owing to the fact both the implementations are the same.

https://github.com/strimzi/strimzi-kafka-operator/blob/23b4ebc6721b90e4b4f9951c0d33d06363fae3e2/docker-images/kafka-based/kafka/scripts/kafka_tls_prepare_certificates.sh#L13

**Comment by scholzj** [comment]
Good catch. Looks like something I missed while refactoring the code. Do you want to open a PR for it? Or should we do it?

**Comment by j4real2208** [comment]
> Good catch. Looks like something I missed while refactoring the code. Do you want to open a PR for it? Or should we do it?

Happy to do the same
